dizoo/classic_control/cartpole/config/cartpole_trex_onppo_config.py

```diff
@@ -10,16 +10,18 @@
     ),
     reward_model=dict(
         type='trex',
+        exp_name='cartpole_trex_onppo_seed0',
```

Comment:
why `exp_name` here

Reply:
in our original implementation, we used exp_name to build the tb logger. So it uses the whole config file. our new implementation only uses the reward model config, so I add this part to the reward model.